obspy/clients/fdsn/client.py

```diff
@@ -45,6 +46,7 @@
 
 DEFAULT_SERVICE_VERSIONS = {'dataselect': 1, 'station': 1, 'event': 1}
 
+URL_REGEX = 'https?://(localhost|\w+(\.\w+){2,})(:\d+)?(/\w+)*/?$'
```

Comment:
This stackoverflow post shows the regex Django uses for that purpose - best just use that: http://stackoverflow.com/questions/7160737/python-how-to-validate-a-url-in-python-malformed-or-not

Passing it through `urlparse` would also work but it would require a couple of additional asserts afterwards so I guess the regex way is easier.